core/src/wallet/stellar/StellarLikeAccount.cpp

```diff
@@ -0,0 +1,524 @@
+/*
+ *
+ * StellarLikeAccount.cpp
+ * ledger-core
+ *
+ * Created by Pierre Pollastri on 13/02/2019.
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2019 Ledger
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ */
+
+#include "StellarLikeAccount.hpp"
+#include <wallet/stellar/StellarLikeWallet.hpp>
+#include <utils/DateUtils.hpp>
+#include <events/Event.hpp>
+#include "database/StellarLikeAccountDatabaseHelper.hpp"
+#include <wallet/common/database/AccountDatabaseHelper.h>
+#include "database/StellarLikeLedgerDatabaseHelper.hpp"
+#include "database/StellarLikeTransactionDatabaseHelper.hpp"
+#include <wallet/common/database/OperationDatabaseHelper.h>
+#include <set>
+#include <wallet/common/BalanceHistory.hpp>
+#include <api/BoolCallback.hpp>
+#include <api/AmountCallback.hpp>
+#include <events/LambdaEventReceiver.hpp>
+#include "transaction_builders/StellarLikeTransactionBuilder.hpp"
+#include <api/BigIntCallback.hpp>
+#include <database/soci-date.h>
+#include <api/StringCallback.hpp>
+#include <api/StellarLikeOperationType.hpp>
+
+
+using namespace ledger::core;
+
+static const std::set<stellar::OperationType> ACCEPTED_PAYMENT_TYPES {
+    stellar::OperationType::PAYMENT, stellar::OperationType::CREATE_ACCOUNT
+};
+
+static const auto INVALID_SYNCHRONIZATION_DELAY_MS = 60 * 60 * 1000;
```

Comment:
I suggest to use anonymous `namespace` instead of `static` keyword. Except If I misunderstood these lines, we don't really care about linkage stuff here.

Reply:
Fixed

Reply:
Fixed by 598be3b60dc85966633aa3082bd808f8744880c0